plugin-server/src/worker/ingestion/event-pipeline/extractHeatmapDataStep.ts

```diff
@@ -29,17 +29,21 @@ export function extractHeatmapDataStep(
     let acks: Promise<void>[] = []
 
     try {
-        const heatmapEvents = extractScrollDepthHeatmapData(event) ?? []
-
-        // eslint-disable-next-line @typescript-eslint/no-floating-promises
-        acks = heatmapEvents.map((rawEvent) => {
-            return runner.hub.kafkaProducer.produce({
-                topic: runner.hub.CLICKHOUSE_HEATMAPS_KAFKA_TOPIC,
-                key: eventUuid,
-                value: Buffer.from(JSON.stringify(rawEvent)),
-                waitForAck: true,
+        const team = await runner.hub.teamManager.fetchTeam(teamId)
```

Comment:
i'll assume this already does caching

Reply:
Yeah it does (probably should be named as such later)